app/scripts/models/state.coffee

```diff
@@ -0,0 +1,32 @@
+root = exports ? this
+
+root.State = class State
```

Comment:
blergh, I have a work in progress refactor that effects this  -- I'll have to redo it... :/

proceed. My fault for doing half-work and not pushing it out :)